src/get-started/install.md

```diff
@@ -45,7 +45,7 @@ use
 
 ```shell
 apt-get install build-essential libsodium-dev libsnappy-dev \
-    librocksdb-dev pkg-config libprotobuf-dev protobuf-compiler
+    pkg-config libprotobuf-dev protobuf-compiler
```

Comment:
But wouldn't it be better to just replace this with  
```
add-apt-repository ppa:exonum/rocksdb
apt-get install build-essential libsodium-dev libsnappy-dev \
    librocksdb5.17 pkg-config libprotobuf-dev protobuf-compiler
```

for simplicity? Users _always_ need the RocksDB, don't they?

Reply:
Here it is removed completely, and the wording below suggests one shall only install it on _some_ Ubuntu distributions.